tasks/main.yml

```diff
@@ -26,7 +26,10 @@
 - name: Install custom includes
   template:
     src: "{{ item }}"
-    dest: "{{ dhcp_config_dir }}/{{ item | basename }}"
+    dest: >-
+      {{ dhcp_config_dir + '/' + (item | basename)
+      if (item | basename | splitext | last) != '.j2' else
+      dhcp_config_dir + '/' + (item | basename | splitext | first) }}
```

Comment:
This can be simplified to:

dest: "{{ dhcp_config_dir }}/{{ ( item | basename ).split('.j2')[0] }}"

Which I think makes it a little more readable.

Reply:
Yes it does.